charts/v2.1/cray-hms-rts/templates/jobs.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.rtsDoInit }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cray-hms-rts-init-deleter
```

Comment:
Should this also have a templated name like the other resoruce?
```
 {{ .Release.Name | default .Values.rtsDefaultName }}-init-deleter
```

Which for the JAWS RTS instance that runs the init job I think would end up with the same name `cray-hms-rts-init-deleter`

Reply:
No, both jobs should only ever get run by 1 chart's deployment and having a set name makes the job get restarted rather than having 2 of them running.